karma.conf.cjs

```diff
@@ -36,6 +36,9 @@ module.exports = function (config) {
     },
     webpack: {
       mode: "production",
+      optimization: {
+        minimize: false,
```

Comment:
did you investigate if `type` could be added to an `import` statement for the library that caused the issue?
Also, can you add a comment in the code of why we are doing this? (ie, issue encountered with library xyz when `minimize` is `true`.

Reply:
I took another look at the issue. Previously my assumption was that it happens because of an interface consumed from `@libp2p/interface-peer-discovery` which turned out to be partially true. After checking build files I can tell that when minimize is enabled it looses a variable exported from the lib. It happens in the `waku_peer_exchange_discovery.ts` with `symbol` variable.

![image](https://user-images.githubusercontent.com/118575614/217653177-f5d187c7-39dc-4b29-a4f4-5802eafaa1f3.png)

It is consistent and for now I didn't find a way around it.

Reply:
What I will check next is - why we have this code in the bundle in the first place.